flake.nix

```diff
@@ -159,7 +159,7 @@
           rust.cargoManifestPath = "cli/Cargo.toml";
         };
         tools = {
-          clang-tools = final.clang-tools_16;
```

Comment:
why downgrade to 15?
we use some clang-format rules only available on >=16.

Reply:
I tried 16 and had failures for both ccls and include-what-you-use that didnt seem to have quick fixes. Upgrading to a newer nixpkgs may help here, but that is a separate PR.

Reply:
Can you check that clang-format is still formatting?
I had some issues with it in the past where it jsut didn't format if it couldn't parse the config file

Reply:
I guess we can just set `breakbeforeconceptdeclarations` to true and downgrade to 15 through. the other setting we were discussing in the comment i linked was added in clang-format 15 actually

Reply:
Yes, it still functions as far as I can tell. Applying clang-format to all of ./pkgdb produces a few changes. Another option here is to keep clang-tools where it is.